src/igvfd/schemas/changelogs/curated_set.md

```diff
@@ -4,6 +4,7 @@
 
 * Rename `assembly` to `assemblies`.
 * Rename `transcriptome_annotation` to `transcriptome_annotations`.
+* Add `external data for catalog` to the enum list of `file_set_type`.
```

Comment:
Reword to "Expand `file_set_type` enum list to include `external data for catalog`." for consistency sake.